pkg/plugins/policies/core/matchers/testdata/matchedpolicies/torules/04.golden.yaml

```diff
@@ -3,21 +3,14 @@ Rules:
 - BackendRefOriginIndex: {}
   Conf:
     http:
-      requestTimeout: 3s
+      requestTimeout: 1s
```

Comment:
That's weird... I'm not sure if it's correct that it changed

Reply:
https://github.com/kumahq/kuma/blob/master/pkg/plugins/policies/core/matchers/testdata/matchedpolicies/torules/04.policies.yaml#L17

Reply:
Should the `kuma.io/zone` be in labels or `inbound.[].tags` ?

Reply:
seems like on both, so should be ok

Reply:
It seems that the change broke something with the test on universal

Reply:
I think it's because of this https://github.com/kumahq/kuma/pull/11449. Could you take a look if it makes sense?